lib/model.js

```diff
@@ -3365,24 +3390,43 @@ class Model {
       if (!options.fields.length) return this;
       if (!this.changed() && !this.isNewRecord) return this;
 
+      const versionFieldName = _.get(this.constructor.rawAttributes[versionAttr], 'field') || versionAttr;
```

Comment:
What is the reason for using `_.get()` here? If you just pass a shallow property name, you could just access it directly